master/buildbot/steps/shell.py

```diff
@@ -200,6 +201,8 @@ def _describe(self, done=False):
             words = self.command
             if isinstance(words, (str, unicode)):
                 words = words.split()
+            elif isinstance(words, Interpolate):
+                return [str(words)]
```

Comment:
I do not sure this solution. Now we can not display just a part of interpolation and in a description, there will be always full version and as I can see, sometimes there is a very long shell command. I think we should consider how to split `Interpolate` command.

@campos-ddc  is it ok right now or we should change it?

Reply:
After talking with Kamil I've changed `str(Interpolate)` to resolve Interpolate with `.getRenderingFor(self.build).result`